ui/src/hosts/constants/index.ts

```diff
@@ -0,0 +1,32 @@
+interface Source {
+  telegraf: string
+  defaultRP: string
+}
+
+interface Value {
+  value: string
+  type: string
+  selected: boolean
+}
+
+interface TempVar {
+  tempVar: string
+  id: string
+  type: string
+  values: Value[]
+}
+
+export const generateTempVarsForHosts = (source: Source): TempVar[] => [
```

Comment:
Would it make since to have this in a utils file? I was a little confused about having a function in constants.

Reply:
we went back and forth on it but decided to put it in constants since it simply generates a tempvar const with a variable input -- thoughts?

Reply:
I would think that it might make sense in src/hosts/utitls/tempvars.js with at name like generateForHosts

Reply:
word